arangod/IResearch/IResearchDataStore.cpp

```diff
@@ -67,15 +68,15 @@ namespace {
 
 class IResearchFlushSubscription final : public FlushSubscription {
  public:
-  explicit IResearchFlushSubscription(TRI_voc_tick_t tick = 0) noexcept
+  explicit IResearchFlushSubscription(uint64_t tick = 0) noexcept
```

Comment:
Why `tick` type is changed?

Reply:
It's uint64_t from rocksdb so I think it should be uint64_t, anyway it's just using

Reply:
That will confuse readers. As in base class we have  TRI_voc_tick_t . So I'd prefer to be consistent with typenames. Even if for now these are the same.

Reply:
The main issue for me rocksdb tick not TRI_vocbase, so I think we should also replace to uint64_t

Reply:
Internally in iresearch also uint64

Reply:
IMO: On server level we operate the abstraction of engine ticks with type "TRI_voc_tick_t". That is not necessary RocksDB ticks (we may replace storage engine in future).  The only requirement is that underlying engine ticks should fit into this type. And all "FlushFeature" related stuff should operate `TRI_voc_tick_t`.